layers/+tools/shell/packages.el

```diff
@@ -226,7 +226,8 @@
       (setq shell-pop-window-position shell-default-position
             shell-pop-window-size     shell-default-height
             shell-pop-term-shell      shell-default-term-shell
-            shell-pop-full-span       shell-default-full-span)
+            shell-pop-full-span       shell-default-full-span
+            shell-pop-in-after-hook   (lambda () (call-interactively #'evil-insert)))
```

Comment:
Shouldn't we hook `evil-insert-state` instead of this lambda?

Reply:
Probably 😅

Reply:
Done